### DISCUSSION TITLE
"password for chatwoot: " on upgrade to new version (Linux VM)

### DISCUSSION THREAD

**Discussion by yairShiloach** [chatwoot/chatwoot#2313] [General]
hi!
I'm trying to upgrade to the new version, but it's asking me to enter my chatwoot password but it does not work when I try to enter the password for my super account.
What password does he mean and how can it be generated?

```
chatwoot@ubuntu-s:~/chatwoot$ sudo cp /home/chatwoot/chatwoot/deployment/chatwoot-web.1.service /etc/systemd/system/chatwoot-web.1.service
[sudo] password for chatwoot: 
Sorry, try again.
[sudo] password for chatwoot: 
Sorry, try again.
[sudo] password for chatwoot: 
sudo: 3 incorrect password attempts
chatwoot@ubuntu-s~/chatwoot$ 
```

**Comment by sojan-official** [comment]
@yairShiloach  if you have installed chatwoot using the installation script please follow 
https://www.chatwoot.com/docs/self-hosted/deployment/linux-vm#upgrading-to-a-newer-version-of-chatwoot

**Reply by yairShiloach** [reply]
Yes - that what I do... its ask for "password for chatwoot: "
and always return "Sorry, try again."

**Reply by sojan-official** [reply]
you wouldn't need the `sudo` ?

**Reply by yairShiloach** [reply]
not working without sudo:
```
Tasks: TOP => db:migrate => db:load_config => environment
(See full trace by running task with --trace)
chatwoot@ubuntu-s-00001:~/chatwoot$ cp /home/chatwoot/chatwoot/deployment/chatwoot-web.1.service /etc/systemd/system/chatwoot-web.1.service
cp: cannot create regular file '/etc/systemd/system/chatwoot-web.1.service': Permission denied
chatwoot@ubuntu-s-00001:~/chatwoot$ sudo cp /home/chatwoot/chatwoot/deployment/chatwoot-web.1.service /etc/systemd/system/chatwoot-web.1.service
[sudo] password for chatwoot: 
Sorry, try again.
[sudo] password for chatwoot: 
Sorry, try again.
[sudo] password for chatwoot: 
```

**Reply by sojan-official** [reply]
if you have used the installation script. the script creates a new user called `chatwoot` and sets up everything under that user.  have you changed any user or directory permissions in this vm ? 

btw you could leave out the cp commands and just to `systemctl restart chatwoot.target` after running the  `rake db:migrate`
